test-apps/display-test-app/vite.config.ts

```diff
@@ -114,6 +115,30 @@ export default defineConfig(() => {
     define: {
       "process.env": process.env, // injects process.env into the frontend
     },
+    resolve: {
+      alias: {
```

Comment:
so this isnt super scalabale, we shouldnt have to hardcode every single new pkg in here
you should be able to programmatically create this, we already kinda do this for the static assets to copy

Reply:
This is still not super scalable. Could you iterate through the dependent packages from package.json as is done for the static assets on line 22? Maybe a simple list of backend packages to exclude could help?